content/en/cloud_cost_management/_index.md

```diff
@@ -192,46 +192,59 @@ The rule stops executing for each resource, once a first match is found. For exa
 
-### Schedule exports
+### Generate cost exports
 
 1. Navigate to [Exports][2] under Azure portal's *Cost Management + Billing*.
-2. Select the billing scope. **Note:** The scope must be *subscription* or *resource group*.
+2. Select the export scope. **Note:** The scope must be *subscription* or *resource group*.
 3. Once the scope is selected, click **Add**.
-4. The metric is **Actual Cost (usage and purchases)**.
-5. Make sure the Export type is `Daily export of month-to-date costs`.
-6. Make sure the File Partitioning is **on**.
-
-Repeat steps one to six with export metric type **Amortized Cost (usage and purchases)**. Datadog recommends using the same storage account container.
-
-### Provide Datadog access to your data
-
-1. Navigate to the Storage Container where your exports are saved.
-    - In the Exports tab, click the link under Storage Account to navigate to it.
-    -  Click the Containers tab.
-    -  Choose the storage container your bills are in.
-2. Select Access Control (IAM) tab.
-3. Click the Role Assignments tab, then click **Add**.  
-4. Choose **Storage Blob Data Reader** and **Cost Management Reader**, then click Next.
-5. Assign these permissions to one of the App Registrations you have connected with Datadog.
-    - To see which App Registrations are connected to Datadog, visit [Datadog's Azure integration][3].
-    -  Click **Select members**, pick the name of the App Registration, and click **Select**.
+
+{{< img src="cloud_cost/exports_scope.png" alt="Select Exports in navigation and choose an export scope"  >}}
+
+4. Select the following Export details:
+  - Metric: **Actual Cost (usage and purchases)**
+  - Export type: **Daily export of month-to-date costs**
+  - File Partitioning: `On`
+  
+{{< img src="cloud_cost/new_export.png" alt="Metric: Actual, Export type: Daily, and File Partitioning: On"  >}}
+
+5. Choose a storage account, container, and directory for the exports.
+6. Select **Create**.
+
+Repeat steps one to six for Metric: **Amortized Cost (usage and purchases)**. Datadog recommends using the same storage container for both exports.
+
+### Provide Datadog access to your exports
+
+1. In the Exports tab, click on the export's Storage Account to navigate to it.
+2. Click the Containers tab.
+3. Choose the storage container your bills are in.
+4. Select the Access Control (IAM) tab.
+5. Click **Add**.  
+6. Choose **Storage Blob Data Reader**, then click Next.
+7. Assign these permissions to one of the App Registrations you have connected with Datadog.
+    - Click **Select members**, pick the name of the App Registration, and click **Select**.
     - Select *review + assign*.
+ 
+If your exports are in different storage containers, repeat steps one to seven for the other storage container.
 
-If your exports are in different storage containers, repeat steps one to five for the other storage containers.
+### Configure Cost Management Reader access
```

Comment:
```suggestion
### Configure cost management reader access
```

Reply:
"Cost Management Reader" is a proper noun (name of the permission) so that's why I had it like this (link: https://learn.microsoft.com/en-us/azure/cost-management-billing/costs/assign-access-acm-data#azure-ea-subscription-scopes) --> What do you think? @estherk15